bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+/**
+ * @TODO: Write JavaDoc comment
```

Comment:
@sigee ah, it is about collaboration in the community. so whenever you leave a TODO in the code, it is hard to track when and who will be working on the TODO items. instead, if there are TODO items, we encourage people to create an issue for the TODO item and link the issue in the code. So in future, if other people read the code and are interested in working on the TODO items, they can find and pickup the issue to work on.